features/step_definitions/sqlStatementsFactory.js

```diff
@@ -133,27 +137,32 @@ function generateSqlStatementsFrom(data) {
             statements: []
         };
 
-        Object.keys(persoon).forEach(key => {
-            let statement;
-            switch(key) {
-                case 'id':
-                    break;
-                case 'inschrijving':
-                    statement = createInsertIntoPersoonslijstStatement(persoon.inschrijving);
-                    break;
-                default:
-                    persoon[key].forEach(p => {
-                        persoonStatements.statements.push(createInsertIntoStatement(key, p));
-                    });
-                    break;
-            }
-
-            if(statement) {
-                persoonStatements.statements.push(statement);
-            }
-        });
-
-        sqlStatements.personen.push(persoonStatements);
+        if(getBsn(persoon) === undefined) {
+            global.logger.info('persoon zonder burgerservicenummer. Geen sql statements generatie', persoon);
+        }
+        else {
+            Object.keys(persoon).forEach(key => {
+                let statement;
+                switch(key) {
+                    case 'id':
+                        break;
+                    case 'inschrijving':
+                        statement = createInsertIntoPersoonslijstStatement(persoon.inschrijving);
+                        break;
+                    default:
+                        persoon[key].forEach(p => {
+                            persoonStatements.statements.push(createInsertIntoStatement(key, p));
+                        });
+                        break;
+                }
+
+                if(statement) {
+                    persoonStatements.statements.push(statement);
+                }
+            });
+
+            sqlStatements.personen.push(persoonStatements);
+        }
     });
```

Comment:
We kunnen dit aanpassen door alleen persoon-type P inserts over te slaan voor personen zonder burgerservicenummer.

```suggestion
Object.keys(persoon).forEach(key => {
      let statement;
      switch (key) {
        case 'id':
          break;
        case 'inschrijving':
          statement = createInsertIntoPersoonslijstStatement(persoon.inschrijving);
          break;
        default:
          persoon[key].forEach(p => {
            if (key === 'persoon' && getBsn(persoon) === undefined) {
              global.logger.info('persoon zonder burgerservicenummer. Geen sql statements generatie voor persoon type P', persoon);
            } else {
              persoonStatements.statements.push(createInsertIntoStatement(key, p));
            }
          });
          break;
      }

      if (statement) {
        persoonStatements.statements.push(statement);
      }
    });

    sqlStatements.personen.push(persoonStatements);

  });
```